src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
>  To keep it simple for now, though, that task can be super minimal and just shell execute NugetPackageVerifier.exe.

👍 

The other alternative is to rely on well known property names until we switch to a task:

```xml
<!-- NuGetPackageVerifier.targets -->
<Target Name="NuGetPackageVerifier">
   <Error Message="_NPVRuleFile not set" Condition="'$(_NPVRuleFile)' = ''" />
   <Error Message="_NPVDirectory not set" Condition="'$(_NPVDirectory)' = ''" />
  <Exec Command="$(MsBuildThisFileDirectory)/NPE.exe --directory $(_NPVDirectory) --file $(_NPVRuleFile)" />
</Target>

<!-- in Coherence-Signed.targets or KoreBuild.targets -->
<PackageDependsOn>$(PackageDependsOn);VerifyPackages
<VerifyPackages>
   <MsBuild Projects="$(MsBuildThisProject)" Properties="$(_NPVRuleFile)=..;$(_NPVDirectory)=..." Targets="NuGetPackageVerifier" />
 </VerifyPackages>
```